### PR TITLE
[ADF-4825] Use ADF-CLI in ADF for versions and latest commit

### PR DESCRIPTION
**Pull request by davidcanonieto**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/ADF-4825
adf-cli scripts fail when they are run locally.
We don't use them in out build

**What is the new behaviour?**
Fix adf-cli scripts to run locally.
Use adf-cli scripts in out build

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/cli/scripts/update-version.ts [hunk 1]
```diff
@@ -98,25 +98,29 @@ function _getLatestVersionFromNpm(tag: string, project: string, logger: logging.
     return latestVersion;
 }
 
-function _updateLibsVersionPerform(path: string, version: string, logger: logging.Logger) {
-    logger.info('Perform libs version...');
+function _updateLibsVersionPerform(args: UpdateArgs, version: string, logger: logging.Logger) {
+    logger.info('Perform libs versioooon...');
```
Comment (DenysVuika) [inline lib/cli/scripts/update-version.ts]: Is that a typooooo?

File: scripts/release/release.sh [hunk 2]
```diff
@@ -33,9 +33,9 @@ release() {
     git checkout development
 
     if $GNU; then
-        ./scripts/update-version.sh -gnu -v $VERSION
+        ./node_modules/@alfresco/adf-cli/bin/adf-cli update-version --pathPackage "$(pwd)" --version $VERSION
     else
-        ./scripts/update-version.sh -v $VERSION
+        ./node_modules/@alfresco/adf-cli/bin/adf-cli update-version --pathPackage "$(pwd)" --version $VERSION --skipGnu true
```
Comment (DenysVuika) [inline scripts/release/release.sh]: Maybe having just one line with `--skipGnu=$GNU` ?

File: scripts/travis/update/update-project.sh [hunk 3]
```diff
@@ -48,9 +48,9 @@ BRANCH="ADF-update-beta-$VERSION"
 git checkout -b $BRANCH
 
 if $GNU; then
```
Comment (DenysVuika) [inline scripts/travis/update/update-project.sh]: You may simplify it by just passing $GNU to the parameter

File: lib/cli/scripts/update-version.ts (hunk 1 above)
Comment (davidcanonieto) [reply lib/cli/scripts/update-version.ts]: hahahaha could beeeee...

File: scripts/release/release.sh [hunk 4]
```diff
@@ -33,9 +33,9 @@ release() {
     git checkout development
 
     if $GNU; then
```
Comment (DenysVuika) [inline scripts/release/release.sh]: you don't need the "if...then" here, just passing $GNU should be enough

**Pull request merged**